ErsatzTV/client-app/src/views/FFmpegProfilesPage.vue

```diff
@@ -1,11 +1,22 @@
 ﻿<template>
     <div>
+        <v-btn color="success" class="ma-4" @click="addRecord()"
+            >Add FFmpeg Profile</v-btn
+        >
         <v-data-table
             :headers="headers"
             :items="ffmpegProfiles"
             :sort-by="['name']"
             class="elevation-1"
         >
+            <template v-slot:[`item.actions`]="{ item }">
+                <v-icon small class="mr-2" @click="editRow(item.id)">
```

Comment:
I think these could be `<v-btn icon>` and the default size (not small) would probably give about the same click target area as the current UI.

Reply:
Done!